decisions/log/0026-async-request-by-default.md

```diff
@@ -0,0 +1,180 @@
+# 26. Switch to async request/response as mandatory (default) behaviour.
+
+Date: 2023-12-05 (updated)
+
+## Status
+
+Proposed
+
+## Context
+
+The current version of the specification[^1] favors synchronous communication over asynchronous communication. As, especially in the early stages of the network, most requests will target not yet existing PF values, the existing approach using synchronous `ListFootprints` calls will not work. PCF data is often not available and is calculated after the request for a specific product arrives at the data provider (Supplier). 
+
+Furthermore managing the access rights upfront for `ListFootprints` requests are complex for large enterprises with different Business Units and many products. Routing requests on specific products is easier than providing PCF data and managing access rights upfront. 
+
+The seconds synchronous request is `GetFootprint` which uses PF IDs to select the requested PCF. But _PF IDs_ are a detail which is usually unknown to the data requester (customer). Data requesters usually only know _Material Identifiers_ (e.g. catalog number, part number).
+Product specific requests would in addition enable data providers (suppliers) to prioritize the calculation of PCF when these are not available yet.
+
+The specification[^1] already adresses all this by supporting `PF Request Events and Responses`. But this feature is _optional_ at the moment whereas synchronous communication mechanisms (as `ListFootprints` or `GetFootprint`) are defined as _mandatory_. This complicates or even block integration of existing PCF exchange apps as those apps usually work in an asynchrounous and productId focused way.
+
+Other exchange networks like Catena-X also use this asynchronous/material approach. With the current specification the PACT API requires the user to handle two APIs with different logic. 
+
+## Decision
+
+The following changes to the tech spec Bikeshed file[^1] shall be made:
+
+1. Change content of `Host System Minimum Requirements` section:
+   * Change `Action Events` from _optional_ to _mandatory_, i.e. move it from _SHOULD_ to _MUST_ paragraph. 
+
+2. Change content of section `Action ListFootprints/Response Syntax`
+    * Add the new HTTP status code 303 to the List of possible `ListStatusCodes`
```

Comment:
To be honest, it depends on how the consumer deals with unknown return codes. ;) This is the reason why I would for sure be more satisfied with a matching 2xx code, as this would signal success and the consumer would be able to ignore the details of the code until it adapts to the new version.. But looks like there is no matching one and we want to tell him "please look over there".